easybuild/easyblocks/l/lammps.py

```diff
@@ -128,12 +127,16 @@ def configure_step(self, **kwargs):
         # cmake has its own folder
         self.cfg['srcdir'] = os.path.join(self.start_dir, 'cmake')
 
-        # Enable following packages, if not configured in easycofig
+        # Enable following packages, if not configured in easyconfig
         default_options = ['BUILD_DOC', 'BUILD_EXE', 'BUILD_LIB', 'BUILD_TOOLS']
         for option in default_options:
             if "-D%s=" % option not in self.cfg['configopts']:
                 self.cfg.update('configopts', '-D%s=on' % option)
 
+        # enable building of shared libraries, if not specified already via configopts
+        if self.cfg['build_shared_libs'] is None and 'BUILD_SHARED_LIBS' not in self.cfg['configopts']:
```

Comment:
Good idea. To reduce false positives (matching previous check):
```suggestion
        if self.cfg['build_shared_libs'] is None and '-DBUILD_SHARED_LIBS=' not in self.cfg['configopts']:
```

Reply:
stricter pattern used in 462e557bf